front/src/pages/Evaluation.tsx

```diff
@@ -17,19 +20,59 @@ interface Model {
   }>
 }
 
+interface FormValues {
+  criterias: {
+    [key: string]: number
+  }
+}
+
 export default function Evaluation() {
   const [evaluation, setEvalaution] = useState<Evaluation>({
-    id: 0,
+    id: 1,
     team_name: 'Leo',
     person: ['Leo'],
   })
+
+  const params = useParams()
+
+  const [criterias, setCriterias] = useState<CriteriasResponse>()
+  const [formValues, setFormValues] = useState<FormValues>({ criterias: {} })
+
+  const loadCriterias = async () => {
+    const criteriasResponse = await getCriterias(1)
+    setCriterias(criteriasResponse)
+  }
+
   useEffect(() => {
     setEvalaution({ id: 1, team_name: 'Nicolas', person: ['Nicolas'] })
-  })
 
-  function handleSubmit(event: FormEvent) {
-    alert('Você clicou no botão')
+    loadCriterias()
+  }, [params])
+
+  function buildRequestPayload(): SendEvaluationScoresPayload {
+    return Object.entries(formValues.criterias).map(([key, value]) => ({
+      evaluatedStudent: 1,
+      evaluatorStudent: 2,
+      evaluatorTeacher: 1,
+      criteriaScore: value,
+      evaluation: evaluation.id,
+      criteria: parseInt(key),
+    }))
+  }
+
+  async function handleSubmit(event: FormEvent) {
     event.preventDefault()
+    const payload = buildRequestPayload()
+    const result = await sendEvaluationScores(payload)
+    console.log(payload)
+  }
+
+  function handleCriteriaScoreChange(event: ChangeEvent<HTMLInputElement>) {
+    const { name, value } = event.target
+    const newFormValues = { ...formValues }
+    const criteriaId = name.split('-')[1]
```

Comment:
:bike: 
Acho assim mais bonito :sweat_smile: 
```suggestion
    const [, criteriaId] = name.split('-')
```